res/controllers/Denon-MC7000.midi.xml

```diff
@@ -92,6 +92,33 @@
                 </option>
             </group>
         </group>
+        <group label="Samplers">
+            <option
+                variable="prevSamplerStop"
+                label="Auto-stop previous samplers when starting new sampler"
+                type="boolean"
+                default="true">
```

Comment:
Does true as default state make sense here? As it alters how Mixxx normaly behaves.

Reply:
This was the previous default, so I figured it would be less surprising to not automatically change it

Reply:
Maybe we can change the default after the next major release, than every existing user has stored the setting and new user get the proper default.

Reply:
Sure!